### PR TITLE
ENH - don't use senstype but coordsys to determine rotation

### PR DESCRIPTION
**Pull request by robertoostenveld**
This solves #1674

### CODE REVIEW

**Comment by robertoostenveld** [comment]
with this change to the code, I can now do this (also included as a test script)

```
%%

elec = [];
elec.label = {
  'Fpz'
  'T7'
  'Cz'
  'T8'
  'Oz'
  };
elec.elecpos = [
  80   0   0 % a
   0  80   0 % l
   0   0  80 % s
   0 -80   0
 -80   0   0
  ];
elec.unit = 'mm';
elec.coordsys = 'eeglab';

%%

cfg = [];
cfg.elec = elec;
layout = ft_prepare_layout(cfg);
ft_plot_layout(layout);

```

and get the expected result.

**Comment by robertoostenveld** [comment]
@arnodelorme, would this solve it when combined with updates to eeglab2fieldtrip?

File: utilities/ft_convert_coordsys.m [hunk 1]
```diff
@@ -525,6 +525,6 @@
 
 T = zeros(4);
 for k = 1:3
-  T(order_out(k),order_in(k)) = flip(k);
+  T(order_in(k),order_out(k)) = flip(k);
```
Comment (schoffelen) [inline utilities/ft_convert_coordsys.m:528]: goed idee buur!

**Review by schoffelen — APPROVED** [review]

**Comment by arnodelorme** [comment]
I have tried the code above and the channels do not seem rotated correctly. I have checked that the "eeglab" keyword was in the coordsys2label.m, so it should all be good.

**Comment by arnodelorme** [comment]
Screen Shot 2021-02-23 at 7 25 28 AM

**Comment by robertoostenveld** [comment]
did you run it with the version from [this branch](https://github.com/robertoostenveld/fieldtrip/tree/issue1674) from the PR? Otherwise it won't work yet.

**Comment by arnodelorme** [comment]
That’s the reason then.

**Comment by arnodelorme** [comment]
The branch is in your personal repo robertoostenveld:issue1674, which means I would need to switch the Git remote etc... Would you mind accepting the changes, and I will double-check in production?

**Comment by robertoostenveld** [comment]
Yes, I'll do that.

**Comment by robertoostenveld** [comment]
however, we do have a blocking issue #1683 at the moment. Until that is resolved, the changes from this PR that I just merged will not be released yet. But they are in the master branch here. 

See here for info https://www.fieldtriptoolbox.org/development/testing/#releasing-the-code

**Comment by arnodelorme** [comment]
Ah OK, I did not know that. I have run your test code and it works for me.

**Pull request merged**